core/modules/replica/ControllerApp.cc

```diff
@@ -50,6 +55,98 @@ string const description =
     " operations in a replication setup.";
 
 using namespace lsst::qserv::replica;
+namespace util = lsst::qserv::util;
+
+void printAsTable(SqlResultSet const& result,
+                  bool sqlShowType) {
+
+    cout << "  error:     " << result.error << "\n"
+         << "  hasResult: " << (result.hasResult ? "yes" : "no") << "\n"
+         << "  #fields:   " << to_string(result.fields.size()) << "\n"
+         << "  #rows:     " << to_string(result.rows.size()) << "\n";
+
+    if (result.hasResult) {
+
+        size_t const numRows = result.rows.size();
+        size_t const numColumns = result.fields.size();
+
+        vector<shared_ptr<vector<string>>> tableDataByColumns;
+        tableDataByColumns.reserve(numColumns);
+
+        for (size_t columnIdx = 0; columnIdx < numColumns; ++columnIdx) {
+            auto tableColumnPtr = make_shared<vector<string>>();
+            tableColumnPtr->reserve(numRows);
+            tableDataByColumns.push_back(tableColumnPtr);
+        }
+        for (auto const& row :  result.rows) {                
+            for (size_t columnIdx = 0; columnIdx < numColumns; ++columnIdx) {
+
+                auto const& field  = result.fields[columnIdx];
+                auto const& cell   = row   .cells [columnIdx];
+
+                auto const& tableColumnPtr = tableDataByColumns[columnIdx];
+                if (row.nulls[columnIdx]) {
+                    tableColumnPtr->push_back("NULL");
```

Comment:
`const` tho. (line 87). I hate that about const shared_ptr.

Reply:
Agree. To avoid this confusion I should probably do:
```
auto& tableColumnPtr = tableDataByColumns[columnIdx];
```

Reply:
Though, this was fixed in a later commit.

Reply:
It could have been `fixup`ed back into this commit.   ...I chose to review by commit because it seemed there was a lot of namespace changing and it was becoming hard to read the whole lot of changes at the same time.

Reply:
Sorry about that! I'm working on improving my `git` commit habits.